Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org' do
-  gem 'cocoapods', '~> 1.11.0'
+  gem 'cocoapods', '~> 1.15.0'
```

Comment:
`bundle exec rake lint:podspec` was failing on macOS 14 with Cocoapods 1.11.0. Updating to the latest version fixed the issue.